.devcontainer/cpp/test/package-managers/cpm/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+if (ENABLE_CPM_TEST)
```

Comment:
Wouldn't you rather have an option(DEVCONTAINER_CPP_BUILD_TEST <default>) in the top-level CMakeLists, similar to how it's done in amp-embedded-infra-lib?

Reply:
In this case, I'd rather not as it serves a different purpose. I don't want to incur the configure overhead of downloading fmtlib here. I'd like to keep this as local and granular as possible. All the tests that incur a configure-time overhead are controlled in this way to keep test execution as fast as possible.

I do see I have swapped the target name from test-* to *-test, that I'll correct.